CMakeLists.txt

```diff
@@ -1,12 +1,12 @@
 cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
 
 
-find_package(IRODS 4.3.0 EXACT REQUIRED)
+find_package(IRODS 5.0.0 REQUIRED)
 set(IRODS_PLUGIN_REVISION "0")
 
 set(CMAKE_C_COMPILER ${IRODS_EXTERNALS_FULLPATH_CLANG}/bin/clang)
 set(CMAKE_CXX_COMPILER ${IRODS_EXTERNALS_FULLPATH_CLANG}/bin/clang++)
-set(CMAKE_EXE_LINKER_FLAGS_INIT "${CMAKE_EXE_LINKER_FLAGS} -stdlib=libc++")
+set(CMAKE_EXE_LINKER_FLAGS_INIT "${CMAKE_EXE_LINKER_FLAGS}")
```

Comment:
Unsure if `CMAKE_EXE_LINKER_FLAGS_INIT` should be set to anything in particular....